code/modules/economy/ATM.dm

```diff
@@ -76,9 +76,41 @@ log transactions
 			to_chat(user, SPAN_INFO("You insert [I] into [src]."))
 			src.attack_hand(user)
 			qdel(I)
+	else if(istype(I, /obj/item/holder))
```

Comment:
you probably shouldn't be able to do this with any holder item

subtype the jones holder off a general holder/cat